components/Aduan/Milestone.vue

```diff
@@ -574,35 +584,36 @@ export default {
       }
       return []
     },
-    showLogSpanLapor (milestone) {
+    showLogSpanLapor(milestone) {
       if (milestone) {
         const logSpanLapor = milestone?.log_span_lapor?.log
         return this.isIdSpanLaporExist(milestone) && logSpanLapor?.length > 0
       }
       return false
     },
-    goToPageFile (fileArray) {
+    goToPageFile(fileArray) {
       if (fileArray.length === 1) {
         const parts = fileArray[0].split('.')
-        const extensionFiles = this.checkExtensionFiles(
-          parts[parts.length - 1]
-        )
+        const extensionFiles = this.checkExtensionFiles(parts[parts.length - 1])
 
-        if (extensionFiles === 'images' || extensionFiles === 'pdf') {
+        if (extensionFiles === 'images') {
           this.$store.commit('setFileAduan', fileArray)
-          this.$router.push({ path: `/aduan-warga/file-aduan/${extensionFiles}`, query: this.$route.query })
-          return
+          this.$router.push({
+            path: `/aduan-warga/file-aduan/${extensionFiles}`,
+            query: this.$route.query,
+          })
+          window.location.href = fileArray[0]
```

Comment:
hello, @rayfajars I just wondering why we changing the URL twice?
1. using router.push
2. using window.location.href 🤔

Reply:
wah thankyou bli kelewat ey, fatal ini mah wkwk

tadi kondisinya untuk file gambar / pdf di push aja

yg file lain pake window.location.href

salah logic nyaa

thankyou blii

[fix here](https://github.com/jabardigitalservice/super-app-web/pull/117/commits/07d3cd0bd1315ce40cc0b4a8475927824f69aae5)

Reply:
wkwkwk sip aman kang ray 👍🏼